clippy_lints/src/lifetimes.rs

```diff
@@ -121,16 +116,10 @@ fn check_fn_inner<'a, 'tcx>(
     report_extra_lifetimes(cx, decl, generics);
 }
 
-fn could_use_elision<
-    'a,
-    'tcx: 'a,
-    T: Iterator<Item = &'tcx Lifetime>
->(
-    cx: &LateContext<'a, 'tcx>,
-    func: &'tcx FnDecl,
-    named_lts: &'tcx [LifetimeDef],
-    bounds_lts: T,
-) -> bool {
+fn could_use_elision<'a, 'tcx: 'a, T: Iterator<Item = &'tcx Lifetime>>(cx: &LateContext<'a, 'tcx>,
```

Comment:
I'm unhappy with the rightwards drift. We could use `fn_args_layout = "Block"`, which would turn it into something like the above if it doesn't fit on one line